spec.bs

```diff
@@ -48,134 +50,162 @@ The following example shows a set of sensor values with [=Snippet=]s and afterwa
 ```
 </div>
 
+The intention of Time Series Snippets is to be a lossless format as, if needed, it can be expanded again. 
+Example 1 for example can be expanded to this RDF dataset:
 
 <div class="example" id="simple_example_rdf">
 ```turtle
-<temperature/1>
+<0>
   a sosa:Observation ;
   sosa:madeBySensor <temp_sensor_1> ;
   sosa:hasSimpleResult "5.4"^^xsd:decimal ;
   sosa:observedProperty: <temperature> ;
-  sosa:resultTime "2024-01-01T06:00:00Z"^^xsd:dateTime .
+  sosa:resultTime "2026-01-01T06:00:00Z"^^xsd:dateTime .
 
-<temperature/2>
+<1>
   a sosa:Observation ;
   sosa:madeBySensor <temp_sensor_1> ;
   sosa:hasSimpleResult "5.2"^^xsd:decimal ;
   sosa:observedProperty: <temperature> ;
-  sosa:resultTime "2024-01-01T06:59:59Z"^^xsd:dateTime .
+  sosa:resultTime "2026-01-01T06:59:59Z"^^xsd:dateTime .
 
-<temperature/3>
+<2>
   a sosa:Observation ;
   sosa:madeBySensor <temp_sensor_1> ;
   sosa:hasSimpleResult "5.2"^^xsd:decimal ;
   sosa:observedProperty: <temperature> ;
-  sosa:resultTime "2024-01-01T08:00:02Z"^^xsd:dateTime .
+  sosa:resultTime "2026-01-01T08:00:02Z"^^xsd:dateTime .
 
-<temperature/4>
+<3>
   a sosa:Observation ;
   sosa:madeBySensor <temp_sensor_1> ;
   sosa:hasSimpleResult "6.1"^^xsd:decimal ;
   sosa:observedProperty: <temperature> ;
-  sosa:resultTime "2024-01-01T09:00:00Z"^^xsd:dateTime .
+  sosa:resultTime "2026-01-01T09:00:00Z"^^xsd:dateTime .
 ```
 </div>
 
 # Definitions # {#definitions}
 
 A <dfn>Time Series</dfn> is a set of data points ordered by timestamp, where each data point consists of a timestamp and corresponding value.
 
-A <dfn>Snippet</dfn> describes a subset of a [=Time Series=] within a certain period defined by a start and end timestamp, and a description of the entity providing the data points.
+A <dfn>Snippet</dfn> describes a subset of a Time Series within a certain period defined by a start and end timestamp, and a description of the entity providing the data points.
 
-A <dfn>Data Point</dfn> is a single point of a [=Time Series=] containing an ISO timestamp, a value with datatype, and optionally an identifier.
+A <dfn>Data Point</dfn> is a single point of a Time Series containing an ISO timestamp, a value with datatype, and optionally an identifier.
 
 # Snippet properties # {#properties}
 
-A [=Snippet=] MUST have the following properties:
+Each Snippet has the following properties:
```

Comment:
Is there a reason why we don't use the RFC standard conventions anymore like `MUST`, `SHOULD`, etc. ?

Reply:
I didn’t change this - I think this must be a merge conflict somehow